Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chromium/Chrome/ChromeBinariesProvider.cs

```diff
@@ -21,7 +21,7 @@
 using System.Net;
 using JetBrains.Annotations;
 
-namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome
+namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium.Chrome
```

Comment:
For WebDriver, Chromium and Chrome are side-by-side, not nested. This has to be symmetrical.